kernel/packages/ui/avatar/chatWindow.ts

```diff
@@ -33,27 +33,18 @@ dcl.onEvent(event => {
   }
 })
 
-function createLogMessage(parent: UIShape, props: { sender: string; message: string; isCommand?: boolean }) {
-  const { sender, message, isCommand } = props
-  const color = isCommand ? COMMAND_COLOR : PRIMARY_TEXT_COLOR
-
-  const messageText = new UIText(parent)
-  messageText.color = color
-  messageText.value = `<b>${sender}:</b> ${message}`
-  messageText.fontSize = 14
-  messageText.vAlign = 'top'
-  messageText.hAlign = 'left'
-  messageText.vTextAlign = 'top'
-  messageText.hTextAlign = 'left'
-  messageText.width = '350px'//330
-  messageText.adaptWidth = false
-  messageText.adaptHeight = true
-  messageText.textWrapping = true
-  messageText.outlineColor = Color4.Black()
+function updateMessagesLog(newMessage: MessageEntry) {
+  messagesLogText.value = ''
+  for (let i = 0; i < internalState.messages.length; i++) {
+    const currentMessage = internalState.messages[i];
+    const color = currentMessage.isCommand ? COMMAND_COLOR : 'white'
+
+    messagesLogText.value += `<color=${color}><b>${currentMessage.sender}:</b> ${currentMessage.message}</color>\n`
+  }
```

Comment:
Can we make this function parameterless? cause with this update we don't need that parameter any more

Reply:
yeah good call